framework/src/org/checkerframework/framework/util/FlowExpressionParseUtil.java

```diff
@@ -116,17 +136,17 @@
      * @param path
      *            The current tree path.
      */
-    public static FlowExpressions. /*@Nullable*/ Receiver parse(String s,
+    public static FlowExpressions. /*@Nullable*/ Receiver parseAllowingItself(String s,
```

Comment:
This is the pattern established throughout many source files that handle flow expressions, and I am only following that pattern. I don't know the reason for this, but I speculate that the name "Receiver" is not clear enough to the reader on its own (it might have been better if the class were called FlowExpressionsReceiver, since it is so widely used). I don't think it makes sense to change this pattern in a few places without changing it everywhere. Would you like me to change FlowExpressions.Receiver to Receiver across all the CF sources?